stdlib/release/src/release/trigger_workflows.bal

```diff
@@ -0,0 +1,125 @@
+import ballerina/config;
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/runtime;
+
+public function main() {
+    http:Client httpClient = new (API_PATH);
+    string accessToken = config:getAsString(ACCESS_TOKEN_ENV);
```

Comment:
Shall we move these two variables to the global scope? So we can avoid them being passed as function params